Sources/HipMobileUI.Droid/HipMobileUI.Droid.csproj

```diff
@@ -368,10 +368,6 @@
       <SubType>Designer</SubType>
     </AndroidResource>
     <AndroidResource Include="Resources\drawable-hdpi\ic_headset_white.png" />
-    <AndroidResource Include="Resources\drawable-mdpi\ic_headset_white.png" />
```

Comment:
Since that icon is actually used (see comment above), these change probably needs to be undone, too.